### PR TITLE
Correct syntactic error

### PR DESCRIPTION
**Pull request by sunlintong**

### CODE REVIEW

File: docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ the machine even if it is being drained of applications in preparation for a reb
 
 ### Node capacity
 
-The capacity of the node (number of cpus and amount of memory) is part of the node object.
+The capacity of the node (numbers of cpu and amount of memory) is part of the node object.
```
Comment (liggitt) [inline docs/concepts/architecture/nodes.md]: revert this change

Comment (steveperry-53) [reply docs/concepts/architecture/nodes.md]: Agree

**Comment by steveperry-53** [comment]
@sunlintong Thanks. After you revert the one change, we can merge this.
